src/styles/_catalog.scss

```
@@ -1,22 +1,30 @@
+$card-size: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(1, $card-size);
```

```suggestion
  display: grid;
  grid-template-columns: repeat(1, $card-size);
  gap: 48px;
  
  justify-content: center;
```

that is not a mistake, but it would be better to use order  https://9elements.com/css-rule-order/